### ISSUE TITLE
Master Crosswalk table for MTL v1.0.

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#54]
Review of the UI itself, for errors or inconsistencies is our task for iteration D testing - not crosswalk tables or the iteration C design document. 

Andrew was to make sure no data-related errors/incosistencies were propagated between the data UI/sim UI, GHPC was reviewing for typos, inconsistencies across the sim UI components.

During this final testing iteration, we need a complete review to find all errors or inconsistencies between the data UI, Vensim model, and in the sim UI, the model diagram, experiment tile data table, experiment tile experiment sliders, output tile, and expanded output tiles, etc. This is especially true, before Takouba finishes MTL 1.0, because everyone else is busy generating content (MTL Live learner guide and MTL Facilitate facilitation guides) that points back to the data UI and pieces of the sim UI. 

We now have had iteration D available for about four weeks, and tracking all that we've found, is even more important than meeting a June 1 deadline. We need critically focused attention on this during our iteration D review/unit 3 testing underway now -  us facilitating frontline teams. And, of course, we need to find all issues, whether anything we find was introduced by programmers since the crosswalk table was produced. Or, whether they are due to Andrew's work and the corrected data pull, etc. 

We will deal with whether/when/how (and costs) of getting any errors/inconsistencies fixed once they have been identified. 
But, at every phase of MTL development, we need all hands on making sure issues and problems don't get carried forward.  
This is true now, and for all the ongoing development work in our immediate future.

To that end, everyone: Please include all of our Team PSD development partners on any threads (I've added everyone). And, please default to our shared platforms for tracking: GitHub and Basecamp.  Quick phone calls with only a few partners, or email threads with attachments and only a part of the team included, is a recipe for confusion and re-work, delays and errors.  Note that some of the rework may not be fully visible to you now, or over a longer collaboration horizon. As an example, work on the Master cross-walk table was handled primarily between Tom and James, in the first place, which is fine. Yet, lots of development time/resources followed from that document, which was principally circulated among only a sub-group of the team. As we continue our work together, be diligent and work to identify seeming short-run time savers, that may be long-run time wasters across the team. And, alert the team with appropriate emphasis ASAP if you see something that could be carrying forward errors in others' workgroups.

We are nearing a finish line on the version 1.0 sim UI development, but everyone's work is interdependent. Therefore, the master crosswalk table for Modeling to Learn 1.0 needs to posted to the GitHub Master branch of Team PSD for reference by all partners producing MTL materials and content. It should, in fact, have been there over the last several months. I've had to try to fish this table out of email for other team members, such as Savet, and this wastes all of our time, and creates risks with version, control, etc.

As with anything on our master branches, this master table of variables, should always be the most current version, accessible for everyone on GitHub. Let's all take responsibility for making sure you cannot make the mistake of moving things forward with a workgroup that bakes in errors or extra work for everyone else. Please join Stacey and I, in recognizing our important this really is!  We welcome all your input, efforts and ideas as we near a critical hand-off of the MTL 1.0 UI.

Stacey: Will you please work with James, Andrew, and Jane and Debbie, today to make sure everyone is using the same, complete, "master" MTL 1.0 cross-walk table to track the errors/inconsistencies that were identified? Everyone the crosswalk folder is on the TeamPSD Master branch and currently empty: https://github.com/lzim/teampsd

Thanks everyone, for all your hard work and diligent, collaborative efforts and expertise!

**Comment by lzim** [comment]
Thanks Everyone!

**Comment by staceypark** [comment]
Hi all,
@holbrooa - Thanks for uploading the file.  Can you clarify why some items are highlighted in yellow?
@branscombj @dlkibbe - Could you give us an update on where we are in the review across the crosswalk and the UI?

**Comment by lzim** [comment]
Thanks All,

Thanks Jane and Debbie for moving this forward next.  Let's all keep in mind in our language the difference between the "simulation UI," the "data UI," and the "master" crosswalk table.
We need to be clear that the data UI is also a "user-interface." And, we need to always track our "masters" - our most complete and correct version of any MTL file - on GitHub.

Please do keep in mind that while reconciling with the simulation UI or "sim UI" is important, the crosswalk file is also being used for the "data UI," which many teams asked to check during their last team meeting. In fact, we have yet to go through and upload a new Andrew-corrected "data UI" and "model parameters" file for each team. Stacey and I have yet to carve out the time to do this task again, but many providers wanted to review before running any experiments in the next team meeting.  

We want to email them soon and point them to these files, but that workflow and stakeholder update is waiting for this review.
Please note that the "master" crosswalk is also used for other data tasks that are on hold until this review is complete.

Therefore, just to make sure we're on the same page, it is great to check the "sim UI" for internal consistency across components (#1). But we need to also track where the "sim UI" and the "master" are different, not just change the master to match the sim UI. Any changes or recommendations that were caught during "sim UI" review/bug reporting to James need to be carried back over to the "data UI" and (if necessary) checked for accuracy by Andrew.  

To restate, please track any changes made to the "master" crosswalk based on "sim UI" feedback reported to James. As those need to be checked for accuracy and updated elsewhere. 
In short, as soon as possible our "master" crosswalk (not the sim UI) needs to be the home base for all the relevant stakeholders, sim UI, data UI, Vensim model files, and all the related interdependent workflows.

Thanks for all your help getting there and recognizing all the forms of interdependencies in parallel workflows!

Lindsey

Note: I've posted this in the relevant GitHub issue where this is being tracked and can be discussed until resolved.

**Comment by lzim** [comment]
Hi All, 

I'd like to close this Master Crosswalk issue to be replaced by the Standardization issue, but it may also be helpful to review the comments on the original issue posted back in May :)

Lindsey
